config/semi_automated.yml

```diff
@@ -449,6 +449,18 @@ parameters:
             tag: 'OAuth CID: 860'
             link: User:Enterprisey/YABBR
             namespaces: [0]
+        effp-helper:
+            regex: '\(\[\[User:Suffusion of Yellow\/effp-helper(\.js)?\|effp-helper\]\]\)'
+            link: User:Suffusion of Yellow/effp-helper
+            namespaces: [0]
+      Reply link:
```

Comment:
Need to indent two more spaces here and on line 460

Reply:
Done